app/views/browse/deployment.html

```diff
@@ -1,141 +1,266 @@
+                <ul class="dropdown-menu actions action-button">
+                  <li ng-if="{ group: 'extensions', resource: 'deployments' } | canI : 'update'">
+                    <a ng-href="project/{{project.metadata.name}}/attach-pvc?kind=Deployment&name={{deployment.metadata.name}}&group=extensions"
+                       role="button">Attach Storage</a>
+                  </li>
+                  <li ng-if="{ group: 'extensions', resource: 'deployments' } | canI : 'update'">
+                    <a ng-href="project/{{projectName}}/set-limits?kind=Deployment&name={{deployment.metadata.name}}&group=extensions"
+                      role="button">Set Resource Limits</a>
+                  </li>
+                  <li ng-if="!autoscalers.length && ({resource: 'horizontalpodautoscalers', group: 'extensions'} | canI : 'create')">
+                    <!-- Create a new HPA. -->
+                    <a ng-href="project/{{projectName}}/edit/autoscaler?kind=Deployment&name={{deployment.metadata.name}}&group=extensions"
+                       role="button">Add Autoscaler</a>
+                  </li>
+                  <li ng-if="autoscalers.length === 1 && ({resource: 'horizontalpodautoscalers', group: 'extensions'} | canI : 'update')">
+                    <!-- Edit an existing HPA. -->
+                    <a ng-href="project/{{projectName}}/edit/autoscaler?kind=HorizontalPodAutoscaler&group=extensions&name={{autoscalers[0].metadata.name}}"
+                      role="button">Edit Autoscaler</a>
+                  </li>
+                  <li ng-if="{ group: 'extensions', resource: 'deployments' } | canI : 'update'">
+                    <a ng-href="{{healthCheckURL}}" role="button">Edit Health Checks</a>
+                  </li>
+                  <li ng-if="{ group: 'extensions', resource: 'deployments' } | canI : 'update'">
+                    <edit-link
+                      resource="deployment"
+                      kind="Deployment"
+                      alerts="alerts">
+                    </edit-link>
+                  </li>
+                  <li ng-if="{ group: 'extensions', resource: 'deployments' } | canI : 'delete'">
+                    <delete-link
+                      kind="Deployment"
+                      resource-name="{{deployment.metadata.name}}"
+                      project-name="{{deployment.metadata.namespace}}"
+                      alerts="alerts"
+                      hpa-list="autoscalers">
+                    </delete-link>
+                  </li>
+                </ul>
+              </div>
+              <small class="meta" ng-if="deployment">created <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></small>
+            </h1>
+            <labels labels="deployment.metadata.labels" clickable="true" kind="deployments" title-kind="deployment configs" project-name="{{deployment.metadata.namespace}}" limit="3"></labels>
```

Comment:
shouldnt title-kind be "deployments" too? in which case i think you can drop title-kind if i'm remembering how the directive works